src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -1,58 +1,209 @@
+```
+
+</Block>
+</BlockSwitcher>
+
+## Initialize a new backend
+
+With the app running, it's time to set up Amplify and create the backend services to support the app.
 
-Amplify.configure(awsconfig);
+Open a new terminal. From the **react-amplified** directory, run:
+
+```bash
+amplify init
+```
+
+When you initialize Amplify you'll be prompted for some information about the project.
+
+<BlockSwitcher>
+<Block name="Expo CLI">
+
+
+<Callout warning>
+
+Note that you have to change some defaults! This will help deployment of the web version.
+
+```console
+? Source Directory Path:  /
+? Distribution Directory Path: web-build
+? Build Command:  npx expo export:web
+? Start Command: npx expo start
+```
+
+</Callout>
+
+```console
+
+? Enter a name for the project (amplifiedtodo) 
+The following configuration will be applied:
+
+Project information
+| Name: amplifiedtodo
+| Environment: dev
+| Default editor: Visual Studio Code
+| App type: javascript
+| Javascript framework: react-native
+| Source Directory Path: src
+| Distribution Directory Path: /
+| Build Command: npm run-script build
+| Start Command: npm run-script start
+
+? Initialize the project with the above configuration? No
+? Enter a name for the environment dev
+? Choose your default editor: Visual Studio Code
+? Choose the type of app that you're building javascript
+Please tell us about your project
+? What javascript framework are you using react-native
+? Source Directory Path:  /
+? Distribution Directory Path: web-build
+? Build Command:  npx expo export:web
+? Start Command: npx expo start
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
+```
+</Block>
+
+<Block name="React Native CLI">
+
+```console
+
+? Enter a name for the project (amplifiedtodo) 
+The following configuration will be applied:
+
+Project information
+| Name: amplifiedtodo
+| Environment: dev
+| Default editor: Visual Studio Code
+| App type: javascript
+| Javascript framework: react-native
+| Source Directory Path: src
+| Distribution Directory Path: /
+| Build Command: npm run-script build
+| Start Command: npm run-script start
+
+? Initialize the project with the above configuration? Yes
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
 ```
 
 </Block>
 </BlockSwitcher>
 
-## Install Amplify libraries
+> Amplify CLI will infer the proper configuration based on the type of project Amplify is being initialized in where possible. Here it detected Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
 
-Next, install the local Amplify dependencies. The directions here will depend on whether you are using Expo or the React Native CLI.
+When you initialize a new Amplify project, a few things happen:
```

Comment:
Thanks for copying this (and the lines below, naturally) over from the React tutorial. I think it is very educational ✨

Reply:
There are a lot of fragments that can be reused as proposed in #5032.